core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1193,6 +1189,15 @@ public int size() {
         };
     }
 
+    /**
+     * Returns the persisted facets.
+     *
+     * @since TODO
```

Comment:
Still todo? This seems like a simple getter :)

Reply:
Hi @afalko, this method is not TODO, this is just a new public API that is being offered, so when it gets released it will automatically be replaced by the Jenkins version (kind of an annotation)